src/drivers/redis.ts

```diff
@@ -55,6 +57,22 @@ export default defineDriver((opts: RedisOptions) => {
   const base = (opts.base || "").replace(/:$/, "");
   const p = (...keys: string[]) => joinKeys(base, ...keys); // Prefix a key. Uses base for backwards compatibility
   const d = (key: string) => (base ? key.replace(base, "") : key); // Deprefix a key
+  // Helper function to scan all keys matching a pattern
+  const scanAllKeys = async (pattern: string): Promise<string[]> => {
+    const client = getRedisClient();
+    const keys: string[] = [];
+    let cursor = "0";
+
+    do {
+      const [nextCursor, scanKeys] = await client.scan(
+        cursor, "MATCH", pattern, "COUNT", batchSize
```

Comment:
The default count size is `10` (less blocking and mem footprint), wondering if we should leave it as as and instead introduce a configuration to customize

Reply:
hey! interesting question. I think to make this customizable in next major version

Reply:
https://github.com/unjs/unstorage/pull/600/commits/39a8617f643d3c00116d391338b1932e9cfcbca9